lib/index.js

```diff
@@ -204,13 +207,9 @@ module.exports.render = function(options) {
   var importer = options.importer;
 
   if (importer) {
-    options.importer = function(file, prev, key) {
+    options.importer = function(file, prev, bridge) {
       function done(data) {
-        console.log(data); // ugly hack
```

Comment:
Ok, I see. Well I am glad to announce that yes this indeed got fixed along with my overhaul :D which I guess is a good thing since the previous hack would not play nice with the `--stdout` option of the cli.

The reason this was happening is because of missed wakeup, i.e. the `notify()` call executed before the `wait()` call, leaving `wait()` hang indefinitely. To work around this, `wait()` has to be guarded by a loop that verifies whether the condition that we are waiting for may have already been met. Because that's a very common pattern `wait()` can actually receive an optional predicate as its second argument that does just that. That's how I used it:

https://github.com/matryo/node-sass/commit/8fdf5c4a85e1d4b135c8e97b7348c826c9ae3a4b#diff-953483bd71622742606ddb26e23f1e62R103